codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/codegen/UnionGenerator.java

```diff
@@ -76,29 +74,4 @@ public void generateUnion(GoWriter writer) {
     private boolean isEventStreamErrorMember(MemberShape memberShape) {
         return isEventStream && memberShape.getMemberTrait(model, ErrorTrait.class).isPresent();
     }
-
```

Comment:
Why was this removed?

Reply:
In Smithy 2.0, the [doc](https://smithy.io/2.0/spec/aggregate-types.html) clearly says 
```
union: The unknown variant. Code generators for unions SHOULD define an unknown variant to account for newly added members.
```

However, when I searched unknown in smithy 1.0 [doc](https://smithy.io/1.0/search.html?q=unknown&check_keywords=yes&area=default), I didn't find any mention of unknown union value. So, I dropped this. However, let me know if you think otherwise.

Reply:
Let's follow the 2.0 guidelines.

Reply:
Done